src/staff.ts

```diff
@@ -10,11 +10,12 @@ import * as db from './db';
  * @return {String} text
  */
 function ticketMsg(name, message) {
-  return `${cache.config.language.dear} `+
-    `${middleware.escapeText(name)},\n\n`+
-    `${middleware.escapeText(message.text)}\n\n`+
-    `${cache.config.language.regards}\n`+
-    `${message.from.first_name}`;
+  const first_name = !cache.config.mod_anonymous ? `${message.from.first_name}` : "";
+  return `${cache.config.language.dear} ` +
+    `${middleware.escapeText(name)},\n\n` +
+    `${middleware.escapeText(message.text)}\n\n` +
+    `${cache.config.language.regards}\n` +
```

Comment:
Please remove the new line here so that the message doesn't have an unnecessary last line.